### ISSUE TITLE
Christmas Cheer Laser Project help Error

### ISSUE THREAD

**Issue by BruRomeo** [PowerShell/PSReadLine#1271]
Error with help command in the laser lab powershell

Environment data
----------------
PS version: 6.2.3
PSReadline version: 2.0.0-beta3
uname : The term 'uname' is not recognized as the name of a cmdlet, function, script file, or oper
able program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try
 again.
At line:23 char:16
+         "os: $(uname -a)"
+                ~~~~~
+ CategoryInfo          : ObjectNotFound: (uname:String) [], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException
 
os: 
PS file version: 6.2.3.0
HostName: ConsoleHost
BufferWidth: 98
BufferHeight: 42

-->

Steps to reproduce or exception report
--------------------------------------
In the Laser Lab Powershell, if you type help you get the below error

PS /home/elf> help
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 5 Keys:
```
 h e l p PageDown
```
Exception:
```
System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.ConsolePal.SetWindowPosition(Int32 left, Int32 top)
   at System.Console.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ScrollDisplayDown(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispat
chTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIn
trinsics, CancellationToken cancellationToken)
```
-----------------------------------------------------------------------
PS /home/elf> help                                                                                
Program 'less' failed to run: No such file or directoryAt line:107 char:21
+ ...     $help | less -Ps"Page %db?B of %D:.\. Press h for help or q to qu ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
At line:107 char:13
+             $help | less -Ps"Page %db?B of %D:.\. Press h for help or ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ResourceUnavailable: (:) [], ApplicationFailedException
+ FullyQualifiedErrorId : NativeCommandFailed

**Comment by msftrncs** [comment]
@BruRomeo, there is a newer version of PSReadLine available, 2.0.0-rc1, this probably will resolve the issue (platform not supported exception).

You are running this on Windows right?  `less` isn't used on Windows.  The `help` command is from PowerShell, so this repository won't be able to help much with the `less` issue, but its interesting.

**Comment by daxian-dbw** [comment]
The `PageDown` exception has been fixed in https://github.com/PowerShell/PSReadLine/pull/1032. Please try out the 2.0.0-rc1 from PowerShell Gallery.

@BruRomeo It looks to me the distro of Linux you are using doesn't have `uname` and `less` commands. The PowerShell function `help` uses `more.com` on Windows and `less` on Linux/macOS to display the help content.
